browser/profiles/brave_renderer_updater.cc

```diff
@@ -64,11 +64,14 @@ BraveRendererUpdater::BraveRendererUpdater(
           &BraveRendererUpdater::CheckActiveWalletAndMaybeUpdateRenderers,
           base::Unretained(this)));
 
+#if BUILDFLAG(ENABLE_WIDEVINE)
+  widevine_enabled_.Init(kWidevineEnabled, local_state);
   local_state_change_registrar_.Init(local_state);
```

Comment:
Do we not need to call this `local_state_change_registrar_.Init(local_state)` if Widevine is not enabled?

Reply:
no, we don't as we use it only to retrieve updates from `kWidevineEnabled` pref

Reply:
I think this is ok because the only registered pref is the widevine one. Maybe should make the declaration conditional too to avoid using the object uninitialized if that changes later?